apps/mobile/src/navigation/linking/__snapshots__/getStateFromPath.test.js.snap

```diff
@@ -0,0 +1,184 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`getStateFromPath ❌ PENDING TEST OUTPUT 1`] = `
+{
+  "/groups/rogue-scholars": {
+    "path": "AuthRoot/Drawer/Tabs/Home Tab/Stream",
+    "routeParams": {
+      "context": "groups",
+      "groupSlug": "rogue-scholars",
+      "originalLinkingPath": "/groups/rogue-scholars",
+      "pathMatcher": "/:context(groups)/:groupSlug",
+    },
+  },
+  "/groups/rogue-scholars/chat/philosophy/post/12345": {
+    "path": "AuthRoot/Post Details - Modal",
+    "routeParams": {
+      "id": "12345",
+      "originalLinkingPath": "/groups/rogue-scholars/chat/philosophy/post/12345",
+      "pathMatcher": ":unmatchedBasePath(.*)/post/:id",
+      "unmatchedBasePath": "/groups/rogue-scholars/chat/philosophy",
+    },
+  },
+  "/groups/rogue-scholars/chat/philosophy/post/12345?commentId=5555": {
```

Comment:
should it be `chats`, not `chat` here?

I have some recollection of fixing that on the navigation url helper... on some branch... but maybe it never made it to dev?

Reply:
Oh yes, the current test cases in there are just a transcription form the spreadsheet, plus some. This should be the place we put any and all routes we want to check and have as our standard tests. If I get to it I'll make a little video about how to utilize this test as it has a couple tricks up its sleeve and more if we need them...